frontend/src/GetDatasetJulia.js

```diff
@@ -44,8 +44,12 @@ export default function GetDatasetJulia({dataset}) {
 # Dataset and column metadata are under dataset_metadata[x] and 
 # column_metadata[x] respectively.
 
+using Pkg
+
+Pkg.add(["HTTP", "JSON", "DataFrames"])
 using HTTP
 using JSON
+using DataFrames
```

Comment:
Can't believe I missed that line - I must have stuck it in the console rather than executing it from the script! 😆

Reply:
Yeah I've been down that road a few times with the REPL as well 😛